universal-login-wallet/src/ui/react/Modals/ModalWaitingFor.tsx

```diff
@@ -20,7 +26,7 @@ const ModalWaitingFor = ({action}: {action: string}) => {
         <div className="modal-pending-section">
           <ProgressBar className="modal-pending-loader"/>
           <h3 className="modal-section-title transaction-hash-title">Transaction status</h3>
-          <p className="txn-hash-text">0x5a63…f3a68a</p>
+          <p className="txn-hash-text"><a href={`https://${chainName.toLowerCase().trim()}.etherscan.io/tx/${transactionHash}`} target="_blank">{transactionHash}</a> </p>
```

Comment:
extract `linkToExplorer(chainName: string, hash:string)`

note: hash can be account or transaction